src/js/components/List/List.js

```diff
@@ -151,8 +153,15 @@ const List = React.forwardRef(
     // If onOrder is not defined but onClickItem is (e.g. the
     // List items are likely selectable), active will be the
     // index of the item which is currently active.
-    const [active, setActive] = useState();
+    const [active, setActiveState] = useState();
```

Comment:
Should we rename `active` to `activeState` ? no change to functionality but we typically have kept these variable names tied together.

Reply:
Changed to leave `setActive` as was and changed new function to be `updateActive`, to make it clear it wasn't just updating a `useState`